stress/common/src/main/java/alluxio/stress/master/MasterBenchTaskResult.java

```diff
@@ -114,7 +93,7 @@ public void setDurationMs(long durationMs) {
    * @return number of successes
    */
   public long getNumSuccess() {
-    return mNumSuccess;
+    return mStatistics.mNumSuccess;
```

Comment:
for these getters and setters which are part of `mStatistics`, do we need them anymore?